### PR TITLE
Added course flag and org unit id attributes to each component

### PR DESCRIPTION
**Pull request by BraydonKains**

### CODE REVIEW

File: src/components/awards.js [hunk 1]
```diff
@@ -39,23 +46,38 @@ class Awards extends BaseMixin(LitElement) {
...
-					text="Course Awards"
-					>
-					<d2l-course-awards class='component'></d2l-course-awards>
-				</d2l-tab-panel>
+				${ this.isCourse ? this._renderCourseTabs() : html`` }
 				<d2l-tab-panel
 					text="My Awards"
 					>
-					<d2l-my-awards class='component'></d2l-my-awards>
+					<d2l-my-awards
```
Comment (quinnbudan) [inline src/components/awards.js:76]: My awards is also a course tab I think

File: src/components/course-awards.js [hunk 2]
```diff
@@ -32,6 +32,10 @@ const AWARD_TYPES = [
 class CourseAwards extends BaseMixin(LitElement) {
 	static get properties() {
 		return {
+			orgUnitId: {
+				attribute: 'org-unit-id',
```
Comment (quinnbudan) [inline src/components/course-awards.js:36]: Can you remove the hardcoded temporary one too and update the API calls to use the value of this property?

File: src/components/my-awards.js [hunk 3]
```diff
@@ -38,6 +38,10 @@ function convertToDateString(dateStr) {
 class MyAwards extends BaseMixin(LitElement) {
 	static get properties() {
 		return {
+			orgUnitId: {
```
Comment (quinnbudan) [inline src/components/my-awards.js:41]: This one also has a hardcoded temporary one

File: src/components/awards.js (hunk 1 above)
Comment (BraydonKains) [reply src/components/awards.js:76]: There will still be a "My Awards" in the org level but it'll comprise of all your awards from different org units

File: src/components/course-awards.js (hunk 2 above)
Comment (BraydonKains) [reply src/components/course-awards.js:36]: Next PR

**Pull request merged**